### PR TITLE
[T] Consistency in underlining in the shared valuation results

### PR DESCRIPTION
**Pull request by majakomel**
[REAL-2751](https://homeday.atlassian.net/browse/REAL-2751)

This PR fixes gaps in dashed leaders that appear with multiline titles: 
Screenshot 2020-08-18 at 14 25 30

### CODE REVIEW

File: src/components/HdDashedList.vue [hunk 1]
```diff
@@ -69,6 +68,7 @@ export default {
 
 .dashed-list {
   $root: &;
+  $leaders-border: 2px dashed #D6DDE6;
```
Comment (kuroski) [inline src/components/HdDashedList.vue:71]:
```suggestion
  $leaders-border: 2px dashed getShade($neutral-gray, 50);
```

I think we are able to use the design system colors here.
https://zeroheight.com/058baf035/p/120a4f-colors/t/44ef55

Then we can achieve the shade values by calling this function:
https://github.com/homeday-de/homeday-blocks/blob/develop/src/styles/_colors.scss#L2

But I might be wrong 😅

**Review by kuroski — COMMENTED** [review]
Great job here =D

**Review by kuroski — APPROVED** [review]
:shipit: Awesome job =D

File: src/components/HdDashedList.vue [hunk 2]
```diff
@@ -69,6 +68,7 @@ export default {
 
 .dashed-list {
   $root: &;
+  $leaders-border: 2px dashed getShade($neutral-gray, 50);;
```
Comment (viniciuskneves) [inline src/components/HdDashedList.vue:71]:
```suggestion
  $leaders-border: 2px dashed getShade($neutral-gray, 50);
```

🚨😂

File: src/components/HdDashedList.vue [hunk 3]
```diff
@@ -129,11 +140,9 @@ export default {
   }
 
   &__dashes {
-    height: 2px;
     flex: 1;
-    background-image: url('~homeday-blocks/src/assets/patterns/dash-pattern-horizontal.svg');
```
Comment (viniciuskneves) [inline src/components/HdDashedList.vue:134]: As far as I've seen you can also delete this asset/folder.

**Review by leandroinacio — APPROVED** [review]

**Pull request merged**